.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
```

Comment:
Do we need PII/email here? Can we just use GitHub aliases?

Reply:
Agree.